pkg/volume/empty_dir/empty_dir.go

```diff
@@ -133,6 +136,7 @@ type emptyDir struct {
 	mountDetector mountDetector
 	plugin        *emptyDirPlugin
 	rootContext   string
+	volume.MetricsDu
```

Comment:
Done. Performed some refactoring to get the path value without an initialized emptyDir.  Also made metricsDu unexported so that it cannot be directly created.  Let me know if you prefer unexported metricsDu, or as it was before.